From: Markus Koschany <apo@debian.org>
Date: Tue, 13 Aug 2019 00:39:57 +0200
Subject: base pom

Quick workaround for the FTBFS with jackson-core 2.9.8. Patch must be dropped
when we update jackson-core again.

Forwarded: not-needed
---
 pom.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/pom.xml
+++ b/pom.xml
@@ -9,7 +9,7 @@
   <parent>
     <groupId>com.fasterxml.jackson</groupId>
     <artifactId>jackson-base</artifactId>
-    <version>2.13.0</version>
+    <version>debian</version>
   </parent>
   <groupId>com.fasterxml.jackson.core</groupId>
   <artifactId>jackson-databind</artifactId>
